src/options.ts

```diff
@@ -111,9 +113,29 @@ export function _setDefaultOptions(options: Partial<Options> = {}): Options {
     ...extraTracerConfig,
   };
 
+  if (options.tracesExporter === undefined) {
+    options.tracesExporter = (process.env.OTEL_TRACES_EXPORTER ||
+      TracesExporter.OTLP) as TracesExporter;
+  }
+
   // factories
-  options.spanExporterFactory =
-    options.spanExporterFactory || defaultSpanExporterFactory;
+  if (options.spanExporterFactory === undefined) {
+    if (options.tracesExporter === TracesExporter.JAEGER) {
```

Comment:
I think this should be purely an env var, not an argument to the function as we already have spanExporterFactory argument.

Reply:
Resolving this because it's the same argument as https://github.com/signalfx/splunk-otel-js/pull/218#discussion_r690266320